Lib9c/Model/Item/Inventory.cs

```diff
@@ -22,6 +24,8 @@ public class Item : IState, IComparer<Item>, IComparable<Item>
         {
             public ItemBase item;
             public int count = 0;
+            public ILock Locked;
+            public bool IsLock => !(Locked is null);
```

Comment:
`.Lock` and `.Locked` are more natural for me.

```suggestion
            public ILock Lock;
            public bool Locked => !(Locked is null);
```

Reply:
I changed the name like this.
`public ILock Locked` => `public ILock Lock`
`public bool IsLock` => `public bool Locked`
`public void Lock(ILock iLock)` => `public void LockUp(ILock iLock)`